### PR TITLE
Fix nightly dirs

### PR DESCRIPTION
**Pull request by Anton-4**

### CODE REVIEW

File: .github/workflows/nightly_linux_x86_64.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 on:
+  workflow_dispatch:
```
Comment (Anton-4) [inline .github/workflows/nightly_linux_x86_64.yml:2]: To allow triggering the workflow manually.

File: .github/workflows/nightly_macos_x86_64.yml [hunk 2]
```diff
@@ -16,7 +17,7 @@ jobs:
         run: ./ci/write_version.sh
           
       - name: execute rust tests
-        run: cargo test --release --locked -- --skip opaque_wrap_function --skip bool_list_literal --skip platform_switching_swift --skip swift_ui --skip str_append_scalar
+        run: cargo test --release --locked -- --skip opaque_wrap_function --skip gen_list::bool_list_literal --skip platform_switching_swift --skip swift_ui --skip gen_str::str_append_scalar --skip gen_tags::phantom_polymorphic_record
```
Comment (Anton-4) [inline .github/workflows/nightly_macos_x86_64.yml:20]: We already skipped this test at [macos_x86_64.yml](https://github.com/roc-lang/roc/blob/main/.github/workflows/macos_x86_64.yml) but I forgot to update it here as well.

File: .github/workflows/test_nightly_macos_apple_silicon.yml [hunk 3]
```diff
@@ -28,7 +29,7 @@ jobs:
         run: ls | grep "roc_nightly.*tar\.gz" | xargs tar -xzvf
 
       - name: delete tar
-        run: ls | grep -v "roc_nightly.*tar\.gz" | xargs rm -rf
+        run: ls | grep "roc_nightly.*tar\.gz" | xargs rm -rf
```
Comment (Anton-4) [inline .github/workflows/test_nightly_macos_apple_silicon.yml:32]: `-v` caused everything but the tar to be deleted.

File: ci/package_release.sh [hunk 4]
```diff
@@ -9,5 +9,15 @@ cp target/release/roc ./roc # to be able to delete "target" later
...
+
+
+mv roc LICENSE LEGAL_DETAILS $1
+
+mkdir $1/examples
+mv examples/helloWorld.roc examples/platform-switching examples/cli $1/examples
+
+mkdir -p $1/crates/compiler/builtins/bitcode
+mv crates/roc_std $1/crates
+mv crates/compiler/builtins/bitcode/src $1/crates/compiler/builtins/bitcode
```
Comment (Anton-4) [inline ci/package_release.sh:21]: e.g. `mv examples/platform-switching $1` used to ditch the examples dir and only put `platform-switching` in the target.

**Review by ayazhafiz — APPROVED** [review]

**Pull request merged**